sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -288,9 +289,13 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
       case PhysicalAggregation(
         namedGroupingExpressions, aggregateExpressions, rewrittenResultExpressions, child) =>
 
+        require(
```

Comment:
Should we throw `AnalysisException` instead?

Reply:
Yeah I was thinking about this one too. I prefer `AnalysisException` too but want to make sure. @cloud-fan what do you think?